packages/unmock-core/src/__tests__/generator.test.ts

```diff
@@ -146,9 +146,11 @@ describe("Tests generator", () => {
     });
     expect(resp).toBeDefined();
     if (resp && resp.body !== undefined) {
-      JSON.parse(resp.body).forEach((pet: any) =>
-        expect(pet.id).toEqual("foo"),
-      );
+      if (!(resp.body instanceof Array)) {
```

Comment:
https://jestjs.io/docs/en/expect.html#tobeinstanceofclass

Reply:
That does not do the type-checking that's necessary for the following `resp.body.forEach`.

Reply:
is it better to use e.g. Array.isArray(resp.body) here? :thinking: 
I know it's relatively new (ES5?) but as it's used elsewhere in the code, I don't see the harm.
It might be more readable?

Reply:
As long as TypeScript understands as type guard 👍

Reply:
And in tests we can use any syntax our test environment uses, I think.

Reply:
Looking at the changes required in this test, I'm afraid it'll be quite painful to use `resp.body` for anything in tests if it's typed as `string | object`... I wonder if it was more user-friendly to do the thing you're not supposed to do and use `any` here for the `body`... 🤔